website/source/docs/guides/deployment.html.md

```diff
@@ -29,33 +27,33 @@ The following instance configurations are recommended.
 
-~> **NOTE** For high workloads, ensure that the disks support high IOPS to keep up with the high Raft log update rate.
+~> **NOTE** For large workloads, ensure that the disks support a high number of IOPS to keep up with the rapid Raft log update rate.
 
-## 1.1 Datacenter Design
+## Datacenter Design
 
-A Consul cluster (typically 3 or 5 servers plus client agents) may be deployed in a single physical datacenter or it may span multiple datacenters. For a large cluster with high runtime reads and writes, deploying servers in the same physical location improves performance. In cloud environments, a single datacenter may be deployed across multiple availability zones i.e. each server in a separate availability zone within a single EC2 instance. Consul also supports multi-datacenter deployments via two separate clusters joined by WAN links. In some cases, one may also deploy two or more Consul clusters in the same LAN environment.
+A Consul cluster (typically 3 or 5 servers plus client agents) may be deployed in a single physical datacenter or it may span multiple datacenters. For a large cluster with a high number of reads and writes, deploying servers in the same physical location improves performance. In cloud environments, a single datacenter may be deployed across multiple availability zones i.e. each server in a separate availability zone within a single EC2 instance. Consul also supports multi-datacenter deployments via two separate clusters joined by WAN links. In some cases, one may also deploy two or more Consul clusters in the same LAN environment.
 
-### 1.1.1 Single Datacenter
+### Single Datacenter
 
-A single Consul cluster is recommended for applications deployed in the same datacenter. Consul supports traditional three-tier applications as well as microservices.
+A single Consul cluster is recommended for applications deployed in the same datacenter.
 
 Typically, there must be three or five servers to balance between availability and performance. These servers together run the Raft-driven consistent state store for catalog, session, prepared query, ACL, and KV updates.
 
 The recommended maximum cluster size for a single datacenter is `5,000` nodes. For a write-heavy and/or a read-heavy cluster, the maximum number of nodes may need to be reduced further, considering the impact of the number and the size of KV pairs and the number of watches. The time taken for gossip to converge increases as more client machines are added. Similarly, the time taken by the new server to join an existing multi-thousand node cluster with a large KV store and update rate may increase as they are replicated to the new server’s log.
```

Comment:
We all agreed that we should soften this wording a little as it has caused some concern. I suggest:

> Consul is proven to work well up to 5000 nodes in a single datacenter gossip pool. Some deployments have stretched this much further but take some care and testing to ensure they remain reliable and converge their cluster state quickly. It's highly recommended that clusters are increased in size gradually when approaching this size.

Reply:
Rephrased in a1961f77c